src/librustdoc/html/static/playpen.js

```diff
@@ -15,6 +15,11 @@ document.addEventListener('DOMContentLoaded', function() {
     'use strict';
 
     if (!window.playgroundUrl) {
+        var run_buttons = document.querySelectorAll(".test-arrow");
```

Comment:
Very very minor, but it looks like naming conventions in the rest of this file use camel case, not snake case.